src/shared/main/java/com/couchbase/lite/AbstractConsoleLogger.java

```diff
@@ -30,11 +30,11 @@
  * The base console logger class.
  */
 abstract class AbstractConsoleLogger implements Logger {
-    private EnumSet<LogDomain> logDomains = LogDomain.ALL_DOMAINS;
-    private LogLevel logLevel = LogLevel.WARNING;
+    private EnumSet<LogDomain> logDomains;
+    private LogLevel logLevel;
 
     // Singleton instance accessible from Database.log.getConsole()
-    AbstractConsoleLogger() { }
+    AbstractConsoleLogger() { reset(); }
```

Comment:
any specific advantage we are getting from calling function reset vs setting as property default values?

Reply:
Yeah.  It guarantees that the initial state is the same as the state to which `reset` sets things, during automated testing...